certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "securityDefinitions": {
+    "basic": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basic": [],
+      "jwt": []
+    }
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "IronNet IronAPI",
+    "description": "IronNet’s IronAPI allows clients to interface with the IronDefense backend through a curated set of endpoints. This provides customers with the capability to triage alerts outside of IronVue.",
+    "version": "3.4.9",
+    "contact": {
+      "name": "IronNet Support",
+      "url": "https://ironnet.com",
+      "email": "support@ironnet.com"
+    },
+    "license": {
+      "name": "Commercial License",
+      "url": "https://ironnet.com"
+    }
+  },
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/IronApi/CommentOnAlert": {
+      "post": {
+        "summary": "Comment on alert",
+        "description": "Allows a client to comment on any given alert, with the option to send to IronDome (if enrolled). The default request rate limit on this endpoint is 10 requests per second.",
+        "operationId": "CommentOnAlert",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/alertCommentOnAlertResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/alertCommentOnAlertRequest"
+            }
+          }
+        ],
+        "tags": [
+          "IronApi"
+        ]
+      }
+    },
+    "/IronApi/GetAlertIronDomeInformation": {
+      "post": {
+        "summary": "Get alert IronDome information",
```

Comment:
@marcus-defazio The summary values should be free from grammatical errors. It can be either `"Get alerts for IronDome information"` or `"Get alerts to get IronDome information"` !!

Reply:
Changed.